docs/source/tutorials/parsl-rp.ipynb

```diff
@@ -0,0 +1,342 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #16.        # RP tasks output located in the pilot sandbox folder
</pre><p>I am not sure the reader will understand what a pilot and a sandbox are in this context as we did not explain either. Note: "P tasks output located " -&gt; "P tasks output is located"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ExaWorks/SDK/pull/156/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>